### PR TITLE
feat: add possibility to add reviewers to the PR

### PR DESCRIPTION
**Pull request by umens**
This PR add the possibility to add a list of reviewers by their username (login for github) or teams to the PR reviewers list.

### CODE REVIEW

**Comment by umens** [comment]
hey @brunobesson 👋  
Do you think you can have a look at this ? 
Thank you for the work here

**Comment by brunobesson** [comment]
Hi there, @umens . Sure, I have seen your PR, thanks for the work. I'll try to handle it ASAP, but it may take a few days. Are you in a hurry to see it merged?

**Comment by umens** [comment]
Thanks for taking the time to reply. No I'm not really in a hurry. If it takes a few days that's fine 🙂

File: README.md [hunk 1]
```diff
@@ -62,6 +64,8 @@ Inputs with defaults are **optional**.
...
 | body           | The body of the pull request.                                                                                                                                                                             | `Caniuse database has been updated. Review changes, merge this PR and have a 🍺.` |
 | labels         | A comma separated list of labels to apply to the pull request.                                                                                                                                            | (no label)                                                                        |
+| reviewers           | A comma separated list of users to be added to the the pull request reviewers list.                                                                                                                                                                           | (no users) |
+| teams           | A comma separated list of teams to be added to the the pull request reviewers list.                                                                                                                                                                          | (no teams) |
```
Comment (brunobesson) [inline README.md:68]:
```suggestion
| reviewers      | A comma separated list of users to be added to the the pull request reviewers list.                                                                                                                                                                           | (no users) |
| teams          | A comma separated list of teams to be added to the the pull request reviewers list.                                                                                                                                                                          | (no teams) |
```

File: action.yml [hunk 2]
```diff
@@ -33,6 +33,12 @@ inputs:
   labels:
     description: Labels to associate to the pull request
     required: false
+  reviewers:
+    description: users to associate to the pull request reviewers list
+    required: false
+  teams:
+    description: teams to associate to the pull request reviewers list
+    required: false
```
Comment (brunobesson) [inline action.yml:41]:
```suggestion
  reviewers:
    description: Users to associate to the pull request reviewers list
    required: false
  teams:
    description: Teams to associate to the pull request reviewers list
    required: false
```

File: src/main.ts [hunk 3]
```diff
@@ -208,6 +211,30 @@ async function run(): Promise<void> {
...
+      .filter((reviewer) => !!reviewer);
+    const teamReviewers = (core.getInput('teams') || '')
+      .split(',')
+      .map((team) => team.trim())
+      .filter((team) => !!team);
+    if (reviewers.length > 0 || teamReviewers.length > 0) {
+      core.info('adding reviewers to the PR');
+      const addReviewersMutationData: AddReviewersMutationVariables = {
+        input: {
+          pullRequestId: browserslistUpdatePR,
```
Comment (brunobesson) [inline src/main.ts:226]: You should not reference `browserslistUpdatePR` there, but `prId` instead. `browserslistUpdatePR` is defined only if there was already an existing PR which then gets updated.

**Review by brunobesson — CHANGES_REQUESTED** [review]
Thanks for your work!

There is an issue with the PR reference, and some minor adjustements I suggest. Other than that I will be happy to include your PR!

File: src/main.ts (hunk 3 above)
Comment (umens) [reply src/main.ts]: You are right sorry :/ I didn't see the error from the compiler the first time. But I just saw it when opening my code editor now. Anyway, it's fixed.

**Review by brunobesson — APPROVED** [review]

**Pull request merged**